src/models/codap-graph-model.ts

```diff
@@ -33,6 +33,7 @@ export const CODAPGraphModel = types.model("ICODAPGraph", {
   primaryAxis: types.maybe(types.string),
   rightSplitAttributeID: types.maybe(types.number),
   rightSplitAttributeName: types.maybe(types.string),
+  showConnectingLines: types.maybe(types.boolean),
```

Comment:
This seems unrelated, just making sure you check that it's intentional.

Reply:
Yep, I just noticed that it was a property I was seeing on graph objects that we didn't have in our model.